api/layout/paths.go

```diff
@@ -40,6 +40,54 @@ func EntriesPathForLogIndex(seq, logSize uint64) string {
 	return EntriesPath(seq/EntryBundleWidth, PartialTileSize(0, seq, logSize))
 }
 
+// RangeInfo represents a range of entries/hashes across one or more entry bundles/tiles.
+type RangeInfo struct {
+	// StartIndex is the index of the first entry bundle the range touches.
+	StartIndex uint64
+	// StartPartial is non-zero if the bundle at StartIndex is expected to be partial.
```

Comment:
Add something that this value references the first index into the bundle? i.e. `(256 * StartIndex) + StartPartial` is the index that this starts from?

At the moment the docs suggest this `StartPartial` is basically a boolean in disguise.